django_loci/base/admin.py

```diff
@@ -256,6 +256,9 @@ def clean(self):
             data['address'] = ''
             data['geometry'] = ''
             data['location_selection'] = 'new'
+        if type_ == 'indoor' and is_mobile:
+            if (data.get('image') or data.get('location')) and ['floor'] not in fields:
+                fields += ['floor']
```

Comment:
I think we could just change the existing code: 

https://github.com/openwisp/django-loci/blob/174a620e3e668095df90b47bcda4468af0a502ae/django_loci/base/admin.py#L249-L253

We could remove `not is_mobile and`, that may indeed be the cause of the bug.